commerce-notification-test/src/testIntegration/java/com/liferay/commerce/notification/test/CommerceNotificationTest.java

```diff
@@ -0,0 +1,368 @@
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerTestRule;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Luca Pellizzon
+ */
+@RunWith(Arquillian.class)
+public class CommerceNotificationTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerTestRule.INSTANCE);
+
+	@Before
+	public void setUp() throws Exception {
+		_company = CompanyTestUtil.addCompany();
+
+		_user = UserTestUtil.addUser(_company);
+
+		_group = GroupTestUtil.addGroup(
+			_company.getCompanyId(), _user.getUserId(), 0);
+
+		_commerceCurrency = CommerceCurrencyTestUtil.addCommerceCurrency(
+			_group.getGroupId());
+
+		_serviceContext = ServiceContextTestUtil.getServiceContext(
+			_company.getCompanyId(), _group.getGroupId(), _user.getUserId());
+
+		_commerceChannelLocalService.addCommerceChannel(
+			_group.getGroupId(),
+			_group.getName(_serviceContext.getLanguageId()) + " Portal",
+			CommerceChannelConstants.CHANNEL_TYPE_SITE, null, StringPool.BLANK,
+			StringPool.BLANK, _serviceContext);
+	}
+
+	@After
+	public void tearDown() throws PortalException {
+		if (_createdAdminRole) {
+			_roleLocalService.deleteRole(_accountAdminRole.getRoleId());
+		}
+
+		if (_createdOrderManagerRole) {
+			_roleLocalService.deleteRole(_orderManagerRole.getRoleId());
+		}
+
+		_commerceNotificationQueueEntryLocalService.
+			deleteCommerceNotificationQueueEntries(_group.getGroupId());
+	}
+
+	@Test
+	public void testAccountAdministratorRecipient() throws Exception {
+		_setUpAccountAdministrator();
+
+		_commerceNotificationTemplate = _setUpNotificationTemplate(
+			"[%ACCOUNT_ROLE_ADMINISTRATOR%]",
+			CommerceOrderConstants.ORDER_NOTIFICATION_PLACED);
+
+		_commerceOrder = CommerceTestUtil.addB2BCommerceOrder(
+			_group.getGroupId(), _user.getUserId(),
+			_commerceAccount.getCommerceAccountId(),
+			_commerceCurrency.getCommerceCurrencyId());
+
+		_commerceNotificationHelper.sendNotifications(
+			_group.getGroupId(), _user.getUserId(),
+			CommerceOrderConstants.ORDER_NOTIFICATION_PLACED, _commerceOrder);
+
+		List<CommerceNotificationQueueEntry> commerceNotificationQueueEntries =
+			_commerceNotificationQueueEntryLocalService.
+				getCommerceNotificationQueueEntries(
+					_group.getGroupId(), QueryUtil.ALL_POS, QueryUtil.ALL_POS,
+					null);
+
+		Assert.assertEquals(
+			commerceNotificationQueueEntries.toString(), 1,
+			commerceNotificationQueueEntries.size());
+
+		CommerceNotificationQueueEntry commerceNotificationQueueEntry =
+			commerceNotificationQueueEntries.get(0);
+
+		String to = commerceNotificationQueueEntry.getTo();
```

Comment:
no need for this variable -- just pass `commerceNotificationQueueEntry.getTo()` to `assertEquals`